File: DailyFlash Solutions/DailyFlash Solutions/54-DailyFlash_Solutions/23_March_Solution_Three/C++/prog5.cpp
#include<iostream>
#include<math.h>
int main() {
	int n, sum = 0, num;
	std::cin >> n;
	int arr[n];
	for(int i = 0; i < n; i++) {
		std::cin >> arr[i];
		sum += arr[i];
	}
	float avg = (float)sum / (float)n;
	sum = 0;
	for(int i = 0; i < n; i++) {
		sum += (avg - arr[i]) * (avg - arr[i]);
	}
	int ch;
        while(1) {
		std::cout << "\n1.Avg\n2.Var\n3.SD\nEnter ch : ";
		std::cin >> ch;
                switch(ch) {
                        case 1:
				std::cout << avg;
                                break;
                        case 2:
				std::cout << (float)sum / n;
                                break;
                        case 3:
				std::cout << sqrt((float)sum / n);
                                break;

                }
        }

}

File: DailyFlash Solutions/DailyFlash Solutions/54-DailyFlash_Solutions/23_March_Solution_Three/Java/prog5.java
import java.util.Scanner;
class Demo {
	public static void main(String[] args) {
		int n, sum = 0, num;
		Scanner sc = new Scanner(System.in);
		n = sc.nextInt();
		int [] arr = new int[n];
		for(int i = 0; i < n; i++) {
			arr[i] = sc.nextInt();
			sum += arr[i];
		}
		float avg = (float)sum / (float)n;
		sum = 0;
		for(int i = 0; i < n; i++) {
			sum += (avg - arr[i]) * (avg - arr[i]);
		}
		int ch;
		while(true) {
			System.out.println("1.avg\n2.var\n3.SD\nch:");
			ch = sc.nextInt();
			switch(ch) {
				case 1:
					System.out.println(avg);
					break;
				case 2:
					System.out.println("Var : " + (float)sum / n);
					break;
				case 3:
					System.out.println("Std dev : " + Math.sqrt((float)sum / n));
					break;
			}
		}
	}
}
